balancer-labs/frontend-v2
# Fix: make sure restaking modal request token approval if needed

# Description

When restaking from old gauge to new gauge, the user needs to approve spend on the new gauge address. This is now included in the restaking modal.

## Type of change


## How should this be tested?

Have a look at a pool where you have balance staked in an old gauge. Remove all approvals for this pool in e.g. https://revoke.cash - then go to restake modal. You should see approval step.

## Visual context

N/A

## Checklist:
